index.js

```diff
@@ -1,5 +1,5 @@
 import { importDeclaration, stringLiteral, isStringLiteral, callExpression } from '@babel/types';
-import { NODE14_BUILTIN_MODULES } from './constants';
```

Comment:
is the file extension needed? shouldn't be the case if it's es6 compatible

Reply:
needed for relative imports https://nodejs.org/api/esm.html#terminology